packages/ui/src/examples/E2_FetchWithStrapiRequest.tsx

```diff
@@ -1,17 +1,37 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { useEffect, useState } from 'react'
-
 import { Box } from '@chakra-ui/react'
+import { strapiRequest } from '@wsvvrijheid/lib';
+import { Blog } from '@wsvvrijheid/types';
 
-import { strapiRequest } from '@wsvvrijheid/lib'
 
 export const FetchWithStrapiRequest = () => {
   const [blogs, setBlogs] = useState([])
```

Comment:
```suggestion
  const [blogs, setBlogs] = useState<Blog[]>([])
```

You can always indicate the type of the state. TypeScript will autocomplete when you type `blog.`